test/transform/split-declaration.js

```diff
@@ -0,0 +1,70 @@
+import createTestHelpers from '../createTestHelpers';
+const {expectTransform} = createTestHelpers(['split-declaration']);
+
+describe('Split Declaration', () => {
+  describe('with only uninitialized variables', () => {
+    it('should split into separate declarations', () => {
+      expectTransform(
+        'var x,y;'
+      ).toReturn(
+        'var x;\nvar y;'
+      );
+    });
+  });
+
+  describe('with uninitialized and initialized variables', () => {
+    it('should split into separate declarations', () => {
+      expectTransform(
+        'var x,y=100;'
+      ).toReturn(
+        'var x;\nvar y=100;'
+      );
+    });
+  });
+
+  describe('with various type of declarations', () => {
+    it('should split into separate declarations', () => {
+      expectTransform(
+        'var x,y=100;let a,b=123;const c=12,d=234'
+      ).toReturn(
+        'var x;\nvar y=100;\nlet a;\nlet b=123;\nconst c=12;\nconst d=234;'
```

Comment:
Lets split the multi-line strings also visually to multiple lines. It would also clarify the input data if it would be broken to multiple lines as well. So something like:

```
      expectTransform(
        'var x,y=100;\n' +
        'let a,b=123;\n' +
        'const c=12,d=234'
      ).toReturn(
        'var x;\n' +
        'var y=100;\n' +
        'let a;\n' +
        'let b=123;\n' +
        'const c=12;\n' +
        'const d=234;'
      );
```